tap_pendo/streams.py

```diff
@@ -406,7 +406,9 @@ def sync_substream(self, state, parent, sub_stream, parent_response):
                             integer_datetime_fmt=
                             "unix-milliseconds-integer-datetime-parsing"
                         ) as transformer:
-                    stream_events = sub_stream.sync(state, new_bookmark,
```

Comment:
Can you please create a new card for this and change this code in a different branch with possible unittest/ integration tests.

Reply:
Raise PR #57 for this bug.